src/plugin/vk-wrapper/src/VkWrapper.cpp

```diff
@@ -40,15 +42,15 @@ void VkWrapper::PrintAvailableExtensions()
     std::vector<VkExtensionProperties> extensions(extensionCount);
     vkEnumerateInstanceExtensionProperties(nullptr, &extensionCount, extensions.data());
 
-    std::cout << "available extensions (" << extensionCount << "):" << std::endl;
```

Comment:
Perhaps you could concatenate the whole list into one message and print it out ?

Reply:
Done